frontend/amundsen_application/static/js/components/Table/index.tsx

```diff
@@ -196,24 +399,45 @@ const Table: React.FC<TableProps> = ({
     isLoading = false,
     numLoadingBlocks = DEFAULT_LOADING_ITEMS,
     rowHeight = DEFAULT_ROW_HEIGHT,
-    expandRow = null,
     emptyMessage,
     onExpand,
     onCollapse,
-    preExpandRow,
+    preExpandPanelKey,
     currentSelectedKey,
+    tableKey,
+    formatChildrenData,
+    preExpandRightPanel,
   } = options;
   const fields = columns.map(({ field }) => field);
   const rowStyles = { height: `${rowHeight}px` };
-  const [expandedRows, setExpandedRows] = React.useState<RowIndex[]>([]);
-  const expandRowRef = React.useRef(null);
+  const expandRowRef = React.useRef<HTMLTableRowElement>(null);
   React.useEffect(() => {
     if (expandRowRef.current !== null) {
-      // @ts-ignore
       expandRowRef.current.scrollIntoView();
     }
   }, []);
 
+  // If the key to preexpand is a nested column, need to add each key level to the expanded row list
+  let keysToExpand: string[] = [];
+  if (preExpandPanelKey) {
+    const columnKeyRegex = tableKey + '\\/(\\w+)';
```

Comment:
Let's move the REGEX part into a constant, adding a comment about what it will match.